course/pages/dagster-essentials/lesson-2/1-set-up-local.md

```diff
@@ -34,46 +36,37 @@ After cloning the Dagster University project, you’ll want to navigate to speci
 cd dagster_university/dagster_essentials
 ```
 
-## Install the dependencies
+## Install uv and dg
 
-**uv**
+Now we want to install `dg`. This is the command line interface that makes it easy to interact with Dagster. Throughout the course we will use `dg` to scaffold our project and streamline the development process.
 
-To install the python dependencies with [uv](https://docs.astral.sh/uv/).
+In order to best use `dg` we will need the Python package manager [`uv`](https://docs.astral.sh/uv/). `uv` will allow us to install `dg` globally and more easily build our virtual environments.
 
+If you do not have `uv` instead already, you can do so with:
 ```bash
-uv sync
+brew install uv
 ```
 
-This will create a virtual environment that you can now use.
-
+Now you can use `uv` to install `dg` globally:
 ```bash
-source .venv/bin/activate
+uv tool install dagster-dg
 ```
 
-**pip**
-
-Create the virtual environment.
+## Install the dependencies
 
+With `uv` and `dg` set, we can create the virtual environment specific to this course. All of the dependencies are maintained in the `pyproject.toml` (you will not need to edit anything in that project for this course). To create the virtual environment, run:
 ```bash
-python3 -m venv .venv
+uv sync
```

Comment:
As mentioned in lesson 3, I think we might need to `uv venv`.

Can you confirm that `uv sync` without it works?

Reply:
Just ran through the workflow and cloning -> going into the subdirectory -> `uv sync` does create the virtual env as well as installing the dependencies